opentracing/go_context.go

```diff
@@ -0,0 +1,27 @@
+package opentracing
+
+import "golang.org/x/net/context"
```

Comment:
this file has all of the even slightly interesting `context.Context` integration code (and there's really not much of it)